src/ExpandablePanel/index.js

```diff
@@ -0,0 +1,222 @@
+// @flow
+import React, { Component } from "react";
+import filter from "lodash/filter";
+import clone from "lodash/clone";
+import classNames from "classnames";
+import withStyles from "react-jss";
+import Paper from "../Paper";
+import CaretRight from "../Icon/CaretRight";
+
+type Props = {
+  children: any,
+  classes: Object,
+  className: string,
+  expanded: boolean,
```

Comment:
Code standards discussion:

Should boolean variables should be prefixed with `is` or `has`?

In this case it would look like `isExpanded`. 

We should decide on how we want to treat booleans in elevate-ui and try to stick with it. Sometimes we use the prefixes and other times we don't:

Prefixed:

https://github.com/elm-street-technology/elevate-ui/blob/develop/src/Button/index.js#L36
https://github.com/elm-street-technology/elevate-ui/blob/develop/src/Textarea/index.js#L24

Not prefixed:

https://github.com/elm-street-technology/elevate-ui/blob/develop/src/Table/CheckboxTable.js#L13
https://github.com/elm-street-technology/elevate-ui/blob/develop/src/MultiSelect/index.js#L25
https://github.com/elm-street-technology/elevate-ui/blob/2152b3b3b1212a4f44019b95845c22788beb78db/src/Button/index.js#L45